### PR TITLE
Add application/graphql support

### LINKED ISSUE #108: Support for Content-Type: application/graphql
Add support for [application/graphql](https://graphql.org/learn/serving-over-http/#post-request) Content-Type. 

express-graphql already supports it, while appolo-server does not.

It is true, that GQL variables can not be passed as part of the query, but it becomes much easier to use in command line, automated tests and [benchmarks](https://github.com/the-benchmarker/graphql-benchmarks).

### PR DESCRIPTION
**Pull request by mihaur**
PR closes #108.

### CODE REVIEW

File: lib/routes.js [hunk 1]
```diff
@@ -151,6 +151,14 @@ module.exports = async function (app, opts) {
     app.route(getOptions)
   }
 
+  app.addContentTypeParser('application/graphql', function (req, done) {
+    var data = ''
+    req.on('data', chunk => { data += chunk })
```
Comment (mcollina) [inline lib/routes.js]: Would you mind to add `req.setEncoding('utf8')`?

File: lib/routes.js [hunk 2]
```diff
@@ -151,6 +151,14 @@ module.exports = async function (app, opts) {
...
+    req.on('end', _ => {
+      done(null, { query: data })
```
Comment (mcollina) [inline lib/routes.js]: How are variables being sent?

Comment (mihaur) [reply lib/routes.js]: Variables are not supported with application/graphql content-type, just query. See https://graphql.org/learn/serving-over-http/#post-request second special case.

curl -H "Content-Type:application/graphql" -XPOST -d "{ add(x: 2, y: 1) }" http://localhost:3000/graphql

Comment (mcollina) [reply lib/routes.js]: Can you add a bit of docs in https://github.com/mcollina/fastify-gql#post-graphql describing this?

Comment (mihaur) [reply lib/routes.js]: I have updated documentation and added new test with mutation via POST and application/graphql.

File: README.md [hunk 3]
```diff
@@ -379,6 +379,28 @@ payload must conform to the following JSON schema:
...
+```js
+{
+  type: 'string',
+  description: 'the GraphQL query'
+}
+```
+
+For code from [example](#example) use:
+```sh
+curl -H "Content-Type:application/graphql" -XPOST -d "{ add(x: 2, y: 2) }" http://localhost:3000/graphql
```
Comment (mcollina) [inline README.md]: This body is not conforming to the description above. It does not respect the JSON schema

Comment (mihaur) [reply README.md]: I think JSON shema was OK. It accepts only a string that is a GraphQL query or mutation.
"{ add(x: 2, y: 1)}" equal to "query { add(x: 2, y: 1)}". Curly braces are part of the query. 

Internally it is transformed to { "query": "query { add(x: 2, y: 1)}" } before execution.

Comment (mcollina) [reply README.md]: A graphql query is not a JSON, so the content of the body cannot follow the JSON schema from a user perspective.

Comment (mihaur) [reply README.md]: Then I will remove the reference to JSON schema validation.

Comment (mihaur) [reply README.md]: I have removed JSON schema validation from POST application/graphql section.

**Comment by mcollina** [comment]
Thanks!

**Pull request merged**